### ISSUE TITLE
Missing file in the train_test_eval_splits.zip

### ISSUE THREAD

**Issue by Malatan** [y2l/social-relation-tensorflow#5]
I realized that 'single_face1_train_16.txt' is missing in the train_test_eval_splits folder which is downloaded in [splits](http://datasets.d2.mpi-inf.mpg.de/qsun_cvpr17/version_1.0/train_test_eval_splits.zip).

**Comment by Malatan** [comment]
And I also trained the model using:
        --train=True 
        --net_arch=vgg19
        --double_stream_mode=True 
        --epoch_num=10 
        --batch_size=10 
        --cls_num=16 
        --learning_rate=1e-4
        --shuffle_dataset=True 
        --img_list1=train_test_eval_splits\annotator_consistency3\single_body1_train_16
        --img_list2=train_test_eval_splits\annotator_consistency3\single_body2_train_16
Is this correct?
I also read in the paper that you used different attributes like age, gender, clothing etc.. for the recognition but I can't find part that is related to this in the code.
